### ISSUE TITLE
B3 repository name

### ISSUE THREAD

**Issue by danielepantaleone** [BigBrotherBot/big-brother-bot#248]
I would like to suggest changing B3 repository name from **big-brother-bot** to simply **b3** since this is the same identifier using on pypi: https://pypi.python.org/pypi/b3/

**Comment by thomasleveil** [comment]
@markweirath any thought on this? 
I'm myself unsure what to think of it. While I don't mind the repo be renamed to just *b3* I'm sure renaming the repo will generate a bit of work load to updating links in the documentation and website.
Having a different name on github and pypi does not bother me either. There is a link to the main B3 website on *b3* module metadata on pypi and from the main website we can find the link to the github repo. What can cheaply be done is to add to the pypi b3 module description the direct link to the github repo.

**Comment by ozon** [comment]
Is this cosmetic change it worth? Links are not working.

**Comment by thomasleveil** [comment]
what links are currently broken?

**Comment by ozon** [comment]
I mean if the name is changed, the left will not work. Is it possible to create an alias?

What show [Google search](https://www.google.de/search?safe=off&site=&source=hp&ei=v7GpVPKhLYXGPbP3gIgP&q=https%3A%2F%2Fgithub.com%2FBigBrotherBot%2Fbig-brother-bot&oq=https%3A%2F%2Fgithub.com%2FBigBrotherBot%2Fbig-brother-bot&gs_l=mobile-gws-hp.12...7908.7908.0.10420.2.2.0.1.1.0.160.268.0j2.2.0.msedr...0...1c.2.60.mobile-gws-hp..1.1.50.3.fzUrHEQfeB8#safe=off&q=https:%2F%2Fgithub.com%2FBigBrotherBot%2Fbig-brother-bot)

**Comment by danielepantaleone** [comment]
GitHub hande repositories redirects: https://github.com/blog/1508-repository-redirects-are-here

**Comment by markweirath** [comment]
Hi guys, I took some time to tinker on this and I can't say that I'm really convinced that the renaming of the repo to b3 has a lot of advantages over the current name.

Did a quick check and I think that Google provides much better results for the current name (google for big-brother-bot or Big Brother Bot) than after changing it to b3 (google for b3 for instance). In fact changing it to b3 would indicate that the product name would be b3 while b3 is just an abbreviation and the product name is still Big Brother Bot... That might even send the wrong signal... Also our homepage domain is www.bigbrotherbot.net. These different facts make me think that we should focus on a strong product name and not on the abbreviation and thus keep the repo name as it is. But, I really am open to argument...

Is there perhaps a programmatic reason for the change of the repository?

**Comment by danielepantaleone** [comment]
Hi Mark

Well there is no reason in particular. I just noticed that on pypi the project name is different and I tought it would be better if the names match. I tried googling for B3, and apart from vitamines :smiley: B3 homepage is 3rd in the first page of the results, but the wikipedia page doesn't show up.
Maybe it's better to do what @thomasleveil suggested, so add the github repository link to the pypi b3 module description.
